### ISSUE TITLE
Log file updates

### ISSUE THREAD

**Issue by ZipCPU** [YosysHQ/sby-gui#14]
## Current behaviour

Last night I ran a run of my [Quad SPI flash cores](https://github.com/ZipCPU/qspiflash).  The first run failed due to a bug in Yosys.  This left behind a log file with an error in it.  (Actually several log files ...)  I then examined the log file, adjusted my code, and re-ran.  This morning, when I looked at the open log file, the same error was shown within it even though the run had completed successfully.

## Expected behavior

The log file in the display window (upper window) should match the current state of the log file as it exists in the directory.  If that means it is a work in progress, then the work in progress log file should be shown.  Similarly, if the log file is open and SymbiYosys finishes, the displayed log file should match the output of SymbiYosys.

In other words, if my test passes, I shouldn't see a log file ending with an ERROR the next morning.
